[docs] Update athena.md (#36199)
metabase/metabase: docs/databases/connections/athena.md

If you are on a paid plan, you can also set cache duration per questions. See [Advanced caching controls](../../configuring-metabase/caching.md#advanced-caching-controls).

## Permissions and IAM Policies

> Most issues that we see when users attempt to connect to AWS Athena involve permissions. Querying AWS Athena requires permissions to:
> - AWS Athena
> - AWS Glue
> - The S3 bucket where Athena results are stored
> - The resources that Athena is querying against (i.e. S3 bucket(s) where Athena is querying)
> - If AWS Lake Formation is used, then AWS Lake Formation permissions must also be granted through the AWS Console (AWS Lake Formation > Permissions > Data Lake Permissions > Grant data lake permissions; the role used by Metabase needs SELECT and DESCRIBE table permissions)

Reviewer: ```suggestion
Most issues that we see when people attempt to connect to AWS Athena involve permissions. Querying AWS Athena requires permissions to:

- AWS Athena.
- AWS Glue.
- The S3 bucket where Athena results are stored.
- The resources that Athena is querying against (i.e., the S3 bucket(s) Athena is querying).
- If you're using AWS Lake Formation, then you also need to grant AWS Lake Formation permissions through the AWS Console (AWS Lake Formation > Permissions > Data Lake Permissions > Grant data lake permissions; the role Metabase uses needs SELECT and DESCRIBE table permissions).
```
